CHANGELOG.adoc

```diff
@@ -19,6 +19,7 @@ https://github.com/oxidecomputer/dropshot/compare/v0.9.0\...HEAD[Full list of co
 
 * https://github.com/oxidecomputer/dropshot/pull/676[#676] changed how TLS configuration is provided to Dropshot.  **`ConfigDropshotTls` is now no longer part of `ConfigDropshot`.**  If you're using TLS, you need to provide this as a separate argument to `HttpServerStarter::new_tls()`.  See #676 for details.
 * https://github.com/oxidecomputer/dropshot/pull/651[#651] The address of the remote peer is now available to request handlers via the `RequestInfo` struct. With this change we've removed the related `From<hyper::Request<B>>` implementation; instead use `RequestInfo::new<B>(&hyper::Request<B>, std::net::SocketAddr)`.
+* https://github.com/oxidecomputer/dropshot/pull/701[#701] adds a new `default_handler_task_mode` field to `ConfigDropshot` and `ServerConfig`. A server configured with `HandlerTaskMode::CancelOnDisconnect` will continue to behave as it has up to this point: a client disconnect will cancel future spawned by its request. `HandlerTaskMode::Detached` will change all handlers to always run to completion, regardless of client disconnects. `ConfigDropshot::default()` will choose `HandlerTaskMode::Detached`, which is a change in behavior if using the default configuration.
```

Comment:
```suggestion
* https://github.com/oxidecomputer/dropshot/pull/701[#701] changes how Dropshot manages the tasks that are used to handle requests.  There are two modes, now configurable server-wide using `HandlerTaskMode`.  Prior to this change, the behavior matched what's now called `HandlerTaskMode::CancelOnDisconnect`: the Future associated with a request handler could be cancelled if, for example, the client disconnected early.  After this change, the default behavior is what's now called `HandlerTaskMode::Detached`, which causes Dropshot to use `tokio::spawn` to run the request handler.  That task will never be cancelled.  This is useful for consumers whose request handlers may not be cancellation-safe.
```

It occurred to me as I was writing this that this might be the first runtime dependency on tokio in Dropshot.  I think when we wrote the bulk of Dropshot, we just assumed consumers were using tokio, but I don't know if we depended in any way on tokio being the executor.  I'm not sure it's possible to use a different executor today.  I don't think it's worth spending much time on this now but at some point we should probably figure out what our dependency really is and document that better.